adapter/redisquota/driver.go

```diff
@@ -37,7 +39,9 @@ type response struct {
 
 // Get is to get a connection from the connection pool.
 func (cp *connPool) get() (*connection, error) {
+	cp.Lock()
```

Comment:
Thanks, Martin. And I found one interesting thing here... even I test memQuota (from the head of master), and run the command-lines, I got the following error:
`Quota RPC returned INVALID_ARGUMENT (builder: name:"default" kind:"quotas" impl:"memQuota" params:&Params{MinDeduplicationDuration:1s,}  aspect: kind:"quotas" adapter:"default" params:&QuotasParams{Quotas:[],} :Unknown quota 'RequestCount' requested), amount 0, expiration 0s`
Do you have any clue of this error?